docs/solution-guidance/security-apponly-azuread.md

```diff
@@ -10,21 +10,21 @@ ms.localizationpriority: medium
 ---
 # Granting access via Azure AD App-Only
 
-When using SharePoint Online you can define applications in Azure AD and these applications can be granted permissions to SharePoint, but also to all the other services in Office 365. This model is the preferred model in case you’re using SharePoint Online, if you’re using SharePoint on-premises you have to use the SharePoint Only model via based Azure ACS as described in [here](security-apponly-azureacs.md).
+When using SharePoint Online you can define applications in Azure AD, which can be granted permissions not only to SharePoint, but also to all the other services in Office 365. This model is the preferred model when using SharePoint Online; if using SharePoint on-premises, you have to use the SharePoint Only model via based Azure ACS, as described [here](security-apponly-azureacs.md).
 
 > [!IMPORTANT]
 > Azure Access Control (ACS), a service of Azure Active Directory (Azure AD), has been retired on November 7, 2018. This retirement does not impact the SharePoint Add-in model, which uses the `https://accounts.accesscontrol.windows.net` hostname (which is not impacted by this retirement). For more information, see [Impact of Azure Access Control retirement for SharePoint Add-ins](https://developer.microsoft.com/office/blogs/impact-of-azure-access-control-deprecation-for-sharepoint-add-ins).
 
 ## Setting up an Azure AD app for app-only access
 
-In Azure AD when doing app-only you typically use a certificate to request access: anyone having the certificate and its private key can use the app and the permissions granted to the app. Below steps walk you through the setup of this model.
+In Azure AD when accessing app-only, you typically use a certificate to request access: anyone having the certificate and its private key can use the app and the permissions granted to the app. Below steps walk you through the setup of this model.
```

Comment:
Incorrect grammar... you don't "access" app-only, you implement or use.

Reply:
@andrewconnell  I was referring to the AAD call. Wouldn't that be an access request?